framework/src/Volo.Abp.Features/Volo/Abp/Features/FeatureInterceptorRegistrar.cs

```diff
@@ -2,14 +2,15 @@
 using System.Linq;
 using System.Reflection;
 using Volo.Abp.DependencyInjection;
+using Volo.Abp.DynamicProxy;
 
 namespace Volo.Abp.Features
 {
     public static class FeatureInterceptorRegistrar
     {
         public static void RegisterIfNeeded(IOnServiceRegistredContext context)
         {
-            if (ShouldIntercept(context.ImplementationType))
+            if (ShouldIntercept(context.ImplementationType) && !DynamicProxyIgnoreTypes.Contains(context.ImplementationType))
```

Comment:
Move `!DynamicProxyIgnoreTypes.Contains(context.ImplementationType)` into the `ShouldIntercept` method (and also in other classes)